.env

```diff
@@ -3,4 +3,4 @@ FLASK_DEBUG=true
 SECRET_KEY=local_development_fake_key
 ENVIRONMENT=devel
 DEVEL=True
-WEBAPP=snapcraft
+WEBAPP=brand
```

Comment:
I think you may have been told about this already but we don't want to commit this.

You can run without modifying this with `./run --env WEBAPP=brand`, or if you aren't using the run script `WEBAPP=brand yarn run serve`

Reply:
@WillMoggridge Yes, sure - we just want to have a PR for iteration demo

Reply:
Aha, OK. Great. :)